opencost/src/settings.tsx

```diff
@@ -0,0 +1,85 @@
+import { NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+
+export function Settings(props) {
+  const { data, onDataChange } = props;
+
+  const settingsRows = [
+    {
+      name: 'Enable in list view',
+      value: (
+        <Checkbox
+          checked={data?.isEnabledInListView}
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+            onDataChange({ ...data, isEnabledInListView: event.target.checked })
+          }
+        />
+      ),
+    },
+    {
+      name: 'Namespace',
+      value: (
+        <TextField
+          fullWidth
+          helperText="Enter the namespace of Opencost service"
```

Comment:
What is this needed for? I think we need more explanation on what this is.

Reply:
Improved the text to `Please enter the namespace where Opencost is Installed`